src/test/features/dashboard/routes/index.ts

```diff
@@ -300,6 +465,7 @@ describe('Dashboard page', () => {
               .get(Paths.dashboardPage.uri)
               .set('Cookie', `${cookieName}=ABC`)
               .expect(res => expect(res).to.be.successful.withText('Your money claims account', 'Make a new money claim'))
+              .expect(res => expect(res).to.be.successful.withText('<a href="/eligibility" class="newclaim" id="start-now" >Make a new money claim</a>'))
```

Comment:
There's no need to check for the anchor link. the line above already checks for the text of the buttton

Reply:
removed this check